.github/workflows/test-integration.yaml

```diff
@@ -4,9 +4,15 @@ on:
   pull_request:
     branches:
       - '*'
+  paths-ignore:
+    - 'docs/**'
+    - '**.md'
```

Comment:
Note sure if the issue is the lack of indentation here, or the duplicate `paths-ignore` declaration
```suggestion
```

Reply:
There were an issue with indentation, according to [documentation](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore). So I fixed it for all workflows. Thanks !